tests/envs/test_registration.py

```diff
@@ -24,11 +26,47 @@ def __init__(self, arg1, arg2, arg3):
 
 
 def test_make():
-    env = envs.make("CartPole-v0")
-    assert env.spec.id == "CartPole-v0"
+    env = envs.make("CartPole-v1")
```

Comment:
Is this also necessary to pass CI all of a sudden?

Reply:
No no it's not, it just removes the warning that the environment is deprecated. Should I keep version 0?

Reply:
Yeah, if you don't mind just leaving it at `-v0`, we can fix things like this in another PR. Same thing with the `isort` changes. These are definitely nice to have but we should keep this PR as the minimal changes needed.